terraform-plans/configs/dcgm-snap_main.tfvars

```diff
@@ -22,7 +22,9 @@ templates = {
   promote = {
     source      = "./templates/github/snap_promote.yaml.tftpl"
     destination = ".github/workflows/promote.yaml"
-    vars        = {}
+    vars = {
+      promote_options = "['core24/edge -> core24/candidate', 'core24/candidate -> core24/stable']"
```

Comment:
should we also include latest/* ? Do we want to keep latest/ and core24/ in sync?

Reply:
I don't know. I guess we should, does it mean that there are going to be 4 options for dcgm?